src/main/java/org/broadinstitute/consent/http/service/EmailService.java

```diff
@@ -180,244 +215,100 @@ private List<String> getMatchingDatasets(Dac dac, List<Dataset> datasetsInDAR) {
   }
 
   private void sendNewDARRequestEmail(
-      User user,
-      Map<String, List<String>> sendList,
-      String researcherName,
-      String darCode
-  ) throws TemplateException, IOException {
-    Writer template = templateHelper.getNewDARRequestTemplate(
-        SERVER_URL,
-        user.getDisplayName(),
-        sendList,
-        researcherName,
-        darCode
-    );
-    Map<String, String> data = retrieveForNewDAR(darCode, user);
-    Optional<Response> response = sendGridAPI.sendNewDARRequests(
-        user.getEmail(),
-        data.get("entityId"),
-        data.get("electionType"),
-        template
-    );
-    saveEmailAndResponse(
-        response.orElse(null),
-        darCode,
-        null,
-        user.getUserId(),
-        EmailType.NEW_DAR,
-        template
-    );
+      User user, Map<String, List<String>> sendList, String researcherName, String darCode)
+      throws TemplateException, IOException {
+    sendMessage(new NewDARRequestMessage(user, darCode, sendList, researcherName), user.getUserId());
```

Comment:
Now that these methods are so short, a future refactor could make `sendMessage()` public and have the service code call `sendMessage()` directly.